playwright-e2e/tests/dsm/participant-page/participant-page-name.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { expect, test } from '@playwright/test';
+import { login } from 'authentication/auth-dsm';
+import { study } from 'pages/dsm/navbar';
+import Select from 'lib/widget/select';
+import ParticipantsPage, { SearchFieldLabel } from 'pages/dsm/participants/participants-page';
+import Table from 'lib/widget/table';
+import { Page } from '@playwright/test';
+
+test.describe('Participant Page DSM', () => {
```

Comment:
you can make this `test.describe.parallel` to run tests in parallel. https://playwright.dev/docs/api/class-test#test-describe-parallel

Reply:
Done 👍